pages/belt_docs.mdx

```diff
@@ -3,33 +3,114 @@ export default Layout;
 
 # Belt
 
-A stdlib shipped with BuckleScript This stdlib is still in beta but we
-encourage you to try it out and give us feedback.
+The preferred standard library for Reason when targeting JavaScript
+(browser, node, React Native).
+
+It is recommended to use the modules from Belt rather than from the
+OCaml standard library or from the Js namespace, e.g. use `Belt.List` instead of
+`List` or `Js.List` and `Belt.Array` instead of `Array` or `Js.Array`.
 
 ## Motivation
 
-The motivation for creating such library is to provide BuckleScript users a
-better end-to-end user experience, since the original OCaml stdlib was not
-written with JS in mind. Below is a list of areas this lib aims to improve:
+Belt improves over the OCaml standard library as follows:
```

Comment:
This is again unsolicited so feel free to ignore, but "improves" might be too strong here too.  Maybe "differs?"  I prefer camel case myself, but not everyone in the community does, so that might not be seen as a general improvement.  Also, not everyone sees pipe-first as an improvement, so that's debatable as an improvement too.

Reply:
Just rewrote the motivation section.